prow.go

```diff
@@ -800,6 +800,9 @@ func (m *jobManager) waitForJob(job *Job) error {
 		}
 		job.PasswordSnippet = strings.TrimSpace(reFixLines.ReplaceAllString(string(logs), "$1"))
 		password, err := commandContents(clusterClient.CoreClient.CoreV1(), clusterClient.CoreConfig, namespace, targetName, "test", []string{"cat", "/tmp/artifacts/installer/auth/kubeadmin-password"})
+		if err != nil {
+			password, err = commandContents(clusterClient.CoreClient.CoreV1(), clusterClient.CoreConfig, namespace, targetName, "test", []string{"cat", "/tmp/shared/installer/auth/kubeadmin-password"})
```

Comment:
this returns the wrong error - you shouldn't overwrite err here because you're hiding the error most jobs return (for an error only some jobs would return)

Reply:
I've updated the logging around these failures.